ores/wsgi/util.py

```diff
@@ -82,3 +84,17 @@ def parse_injection(request, rev_id):
         return {rev_id: cache} if len(cache) > 0 else None
     except Exception as e:
         raise CacheParsingError(e)
+
+
+def nocache(route):
+    @wraps(route)
+    def no_cache(*args, **kwargs):
+        response = make_response(route(*args, **kwargs))
+        response.headers['Cache-Control'] = \
+            "no-store, no-cache, must-revalidate, post-check=0, " + \
+            "pre-check=0, max-age=0"
+        response.headers['Pragma'] = 'no-cache'
```

Comment:
this is redundant these days. Feel free to keep it in just to satisfy really really old clients but I doubt these clients would be accessing ORES (and even more having caching issues)